frontend/src/pages/blocks/[hash].jsx

```diff
@@ -55,46 +57,52 @@ class BlockDetail extends React.Component {
     };
 
     return (
-      <>
-        <Head>
-          <title>NEAR Explorer | Block</title>
-        </Head>
-        <Content
-          title={
-            <h1>{`Block ${
-              block.height
-                ? `#${block.height}`
-                : `${block.hash.substring(0, 7)}...`
-            }`}</h1>
-          }
-          border={false}
-        >
-          {this.props.err ? (
-            `Information is not available at the moment. Please, check if the block hash is correct or try later.`
-          ) : (
-            <BlockDetails block={block} />
-          )}
-        </Content>
-        {!this.props.err ? (
+      <Translate>
+        {({ translate }) => (
           <>
+            <Head>
+              <title>NEAR Explorer | Block</title>
+            </Head>
             <Content
-              size="medium"
-              icon={<TransactionIcon style={{ width: "22px" }} />}
-              title={<h2>Transactions</h2>}
+              title={
+                <h1>{`${translate("page.blocks.title").toString()} ${
+                  block.height
+                    ? `#${block.height}`
+                    : `${block.hash.substring(0, 7)}...`
+                }`}</h1>
+              }
+              border={false}
             >
-              <Transactions blockHash={block.hash} count={1000} />
+              {this.props.err ? (
+                `Information is not available at the moment. Please, check if the block hash is correct or try later.`
+              ) : (
+                <BlockDetails block={block} />
+              )}
             </Content>
+            {!this.props.err ? (
+              <>
+                <Content
+                  size="medium"
+                  icon={<TransactionIcon style={{ width: "22px" }} />}
+                  title={
+                    <h2>{translate("common.transactions.transactions")}</h2>
```

Comment:
Translation component? (I see a few other places in pages/ folder where it could be used)